### ISSUE TITLE
Terms of Service

### ISSUE THREAD

**Issue by michaelcpuckett** [mayeaux/nodetube#123]
Wondering if the terms of service was copied/generated from something specific for open source projects?

Specifically wondering if this line makes sense:

> you may not ... disassemble, decompile or reverse engineer any of the software components of the Service,

**Comment by mayeaux** [comment]
Yeah that can be removed from TOS, I will give it a once over and see if anything else needs to go too to reflect that it's FOSS.
